tutorials/inputs/input_examples.rst

```diff
@@ -0,0 +1,244 @@
+.. _doc_input_examples:
+
+Input Examples
+==============
+
+Introduction
+------------
+
+In this tutorial, you'll learn how to use Godot's :ref:`InputEvent <class_InputEvent>`
+system to capture player input. There are many different types of input your
+game may use - keyboard, gamepad, mouse, etc. - and many different ways to
+turn those inputs into actions in your game. This document will show you some
+of the most common scenarios, which you can use as starting points for your
+own projects.
+
+.. note:: For a detailed overview of how Godot's input event system works,
+          see :ref:`doc_inputevent`.
+
+Events versus polling
+---------------------
+
+Sometimes you want your game to respond to a certain input event - pressing
+the "jump" button, for example. For other situations, you might want something
+to happen as long as a key is pressed, such as movement. In the first case,
+you can use the ``_input()`` function, which will be called whenever an input
+event occurs. In the second case, Godot provides the :ref:`Input <class_Input>`
+singleton, which you can use to query the state of an input.
+
+Examples:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    # input event - runs when the input happens
+    func _input(event):
+        if event.is_action_pressed("jump"):
+            jump()
+
+    # polling - runs every frame
+    func _process(delta):
```

Comment:
```suggestion
    func _physics_process(delta):
```

_process runs at a variable rate, independent from framerate. (You can get 20 process ticks at 60fps. With _physics_process still ticking at 60.) If this happens, the original code would introduce stutter.

All motion should occur in _physics_process. Let's not show newbies code that has the potential of breaking down later.

Reply:
Done.